### ISSUE TITLE
how can i use the hotspot functions of videojs-panorama?

### ISSUE THREAD

**Issue by ssalbal** [yanwsh/videojs-panorama#117]
Can i use hotspot text and image.
Panorama HotSpot Support description. How can i use the hotspot function.?

**Comment by yanwsh** [comment]
yes,

Here are the example code, please download beta version [here](https://github.com/yanwsh/videojs-panorama/releases/tag/v1.0.0-beta). You can use both image and text,

For image, please parse dom to element. 

Like 

```
 element:  document.getElementById("hotspot")

```

```
 Markers: [
                {
                    location: {
                        lat: 0,
                        lon: 180
                    },
                    radius: 500,
                    element: "This is text 1 with long text"
                },
                {
                    location: {
                        lat: 20,
                        lon: 160
                    },
                    radius: 500,
                    element: "This is text 2 with long text",
                    onShow: function(){
                        console.log("text 2 is shown");
                    },
                    onHide: function(){
                        console.log("text 2 is hidden");
                    }
                }
            ]
```

**Comment by ssalbal** [comment]
hotspot does not same to Markers.
could you more detail example code?

**Comment by yanwsh** [comment]
Could you explain how does it looks like?

**Comment by ssalbal** [comment]
I just insert element & Markers to player.panorama options like below code.
But, no text find on the video.  I have to write more html5 hotspot code?

        player.panorama({
            clickToToggle: (!isMobile()),
  
            clickAndDrag: true,
            autoMobileOrientation: true,
  
            initFov: 100,  
            VREnable: isMobile(),

            NoticeMessage: (isMobile())? "please drag and drop the video" : "please mouse drag and drop the video",
            autoHideNotice: 3000, 

            element:  document.getElementById("hotspot"),
            Markers: [
                {
                    location: {
                        lat: 0,
                        lon: 180
                    },
                    radius: 500,
                    element: "This is text 1 with long text"
                },
                {
                    location: {
                        lat: 20,
                        lon: 160
                    },
                    radius: 500,
                    element: "This is text 2 with long text",
                    onShow: function(){
                        console.log("text 2 is shown");
                    },
                    onHide: function(){
                        console.log("text 2 is hidden");
                    }
                }
            ],

**Comment by yanwsh** [comment]
Does it showing any console error?

**Comment by ssalbal** [comment]
Yes. thereis no console error.

Just Got Warning Console log.  VIDEOJS: WARN: videojs.plugin() is deprecated; use videojs.registerPlugin() instead.

assets/shark.mp4 panorama view well.

I'm using 
video.js ( Ver. 6.2.1) / three.js (Ver r75) / videojs-panorama.v5.js (Ver 1.0.0-beta)

Also tested with video.js(Ver 5.20.1).  the result same - no hotspot text. no console error.

**Comment by ssalbal** [comment]
Also I'm using  O.S - Windows 7  , Browser - Chrome (Ver 59.0.3071.115)

**Comment by yanwsh** [comment]
Could you send me your project files or send me a demo, so i can figure out what's wrong.

**Comment by ssalbal** [comment]
yes. demo link [here](http://b2i.co.kr:7002/webVR/vjs-demo-panorama/videojs-panorama/index.html)

It's just some modified index.html of videojs-panorama original file.

**Comment by yanwsh** [comment]
Hey, looks like you are not use beta version. Only beta version support hotspot right now. Please download code [here](https://github.com/yanwsh/videojs-panorama/releases/tag/v1.0.0-beta).

**Comment by ssalbal** [comment]
it's my mistaken.. it is well display text hotspot with beta version.
could you let me know, how to control hotspot "text size" "text color" and display image with text.
also can i use clickable image with url link?

**Comment by yanwsh** [comment]
If you right click and open console, you can see all the text showing on video is html dom, so you can add css code to change it.

```
        .vjs-marker{
            white-space: normal;
            width: 100px;
            font-size: 2em;
        }
```

**Comment by ssalbal** [comment]
it was really big helpness to me. thanks to your help.

**Comment by Vishnuvarth** [comment]
Can you provide any example of working with image hotspot...!
